content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,130 @@
+distributed traces and metrics that help explain performance behaviors and accelerate root cause analysis. For more
+information, visit [LightStep](https://lightstep.com).
+At the end of this task, trace spans will be sent from the proxies, to a LightStep [𝑥]PM Satellite pool, and then be
+consumable in the web UI.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample is used as the
+example application for this task.
+
+## Before you begin
+
+> {{< idea_icon >}} The following instructions assume you have a LightStep account and a Satellite pool configured with
+TLS certs and a secure GRPC port exposed. To create an account, [contact LightStep](https://lightstep.com/contact/).
+For details about setting up Satellites, see [LightStep Satellite Setup](https://docs.lightstep.com/docs/satellite-setup).
+
+1.  Make sure that you have a LightStep Access Token and that the Satellite pool can be reached at an address in the
+    form of `<Host>:<Port>` (ex. `lightstep-satellite.lightstep:9292`).
+1.  Follow [the Helm install instructions](/docs/setup/kubernetes/helm-install/) to install Istio via Helm.
+    You must pass a few arguments using the `--set key=value` syntax when you run the `helm` command, for example:
+
+    {{< text bash >}}
+    $ helm template \
+        --set global.proxy.tracer="lightstep" \
+        --set global.tracer.lightstep.address="<Satellite Address>" \
+        --set global.tracer.lightstep.accessToken="<LightStep Access Token>" \
+        --set global.tracer.lightstep.secure=true \
+        --set global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem" \
+        install/kubernetes/helm/istio \
+        --name istio --namespace istio-system > $HOME/istio.yaml
+    $ kubectl create namespace istio-system
+    $ kubectl apply -f $HOME/istio.yaml
+    {{< /text >}}
+
+1.  Create a secret in the default namespace (or whichever namespace you are deploying the Bookinfo application into)
+    with the Certificate Authority certificate used for verifying TLS communication with the Satellite pool.
+
+    {{< text bash >}}
+    $ CACERT=$(cat Cert_Auth.crt | base64) # Cert_Auth.crt contains the necessary CACert
+    $ NAMESPACE=default
+    {{< /text >}}
+
+    ```bash
+    $ cat <<EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: lightstep.cacert
+        namespace: $NAMESPACE
+        labels:
+          app: lightstep
+      type: Opaque
+      data:
+        cacert.pem: $CACERT
+    EOF
+    ```
+
+1.   Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+## Visualize trace data
+
+1.  Send traffic to the sample application.
+
+    For the
+    [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port)
+    sample, visit `http://$GATEWAY_URL/productpage` in your web browser
+    or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
```

Comment:
If I run the command after following the commands in this document the command fails. Nowhere is the `$GATEWAY_URL` variable exported.